sql/mysql_db/user.go

```diff
@@ -70,6 +75,11 @@ func UserToRow(ctx *sql.Context, u *User) (sql.Row, error) {
 	if u.Attributes != nil {
 		row[userTblColIndex_User_attributes] = *u.Attributes
 	}
+	row[userTblColIndex_ssl_type] = u.SslType
+	row[userTblColIndex_ssl_cipher] = []uint8(u.SslCipher)
+	row[userTblColIndex_x509_issuer] = []uint8(u.X509Issuer)
+	row[userTblColIndex_x509_subject] = []uint8(u.X509Subject)
```

Comment:
It feels like we should note why this is a byte slice rather than just adding the string. Not sure _where_ the note should go (since this also applies in `UserFromRow`), but it's different enough from the other fields that it seems like it should be documented. It's not major if we don't, but just for future documentation in case someone comes across this.